portalnet/src/events.rs

```diff
@@ -103,3 +103,10 @@ impl PortalnetEvents {
         }
     }
 }
+
+/// Events that can be produced by the `OverlayProtocol` event stream.
+#[derive(Debug, Clone, PartialEq, Eq)]
+pub enum OverlayEvent {
+    LightClientOptimisticUpdate,
```

Comment:
We can expand those updates by including more data like timestamps, content data and etc., but I'm keeping this simple for now until we know what we want exactly.

Reply:
Thoughts on implementing a simple "envelope" struct like this to go ahead and include timestamps.

```
struct EventEnvelope:
    timestamp: DateTime
    payload: OverlayEvent
```

Reply:
I like it, implemented in https://github.com/ethereum/trin/pull/793/commits/63c77de1453f62765b3cb999f3b1174ea608ffa8.